draft-ietf-gnap-core-protocol.md

```diff
@@ -5748,15 +5841,20 @@ and abused in making needlessly insecure systems.
 
 In GNAP, key-bound access tokens are the default due to their higher security properties. While
 bearer tokens can be used in GNAP, their use should be limited to cases where the simplicity
-benefits outweigh the significant security downsides.
+benefits outweigh the significant security downsides. One common deployment pattern is to use a
```

Comment:
Maybe (to make it more concrete) we could note that with advanced MFA becoming more mainstream (e.g. FIDO2/webauthn/keypass), the next frontier will be token theft. Therefore having some kind of protection is important